src/components/PartyDropdown.js

```diff
@@ -0,0 +1,149 @@
+import React from "react"
+import { parties, partyLogo } from "../models/information"
+import { labelColor } from "../styles"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faSearch } from "@fortawesome/free-solid-svg-icons"
+
+export default ({ open }) => {
+  function partyItem(p) {
+    return (
+      <div
+        css={{
+          display: "grid",
+          gridTemplateColumns: "30px auto",
+        }}
+      >
+        <img
+          alt=""
+          src={partyLogo(p.name)}
+          css={{
+            maxHeight: "100%",
+            maxWidth: "100%",
+            width: "auto",
+            height: "auto",
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            margin: "auto",
+          }}
+        />
+        <div css={{ fontSize: "2em", paddingLeft: 15 }}>
+          <b>{p.name}</b>
+        </div>
+      </div>
+    )
+  }
+
+  function renderDefaultDropdown() {
+    return (
+      <div
+        css={{
+          cursor: "pointer",
+        }}
+        onClick={() => {
+          // open = true
```

Comment:
put a comment with `@todo`for onClick here ka 
it will create issue and make sure we wont miss a thing :)